build.fsx

```diff
@@ -146,7 +146,8 @@ Target "Build" (fun _ ->
 Target "RunTests" (fun _ ->
     !! testAssemblies
     |> NUnit3 (fun p ->
-        { p with TimeOut = TimeSpan.FromMinutes 20.})
+        { p with TimeOut = TimeSpan.FromMinutes 20.
+                 ProcessModel = NUnit3ProcessModel.SingleProcessModel })
```

Comment:
Not sure if you want this change - but I couldn't get the tests to run otherwise on my machine.